docs/data_pack.md

```diff
@@ -1,5 +1,31 @@
 # DataPack Tutorial
 
+## Get primitive data
+`DataPack.get_data()` is commonly used to retrieve primitive data from a `DataPack`. User can request particular data fields within the range of a particular `Annotation` or `AudioAnnotation` type. User request particular data fields by setting `request` and the search range by setting `context_type`.
+
+In forte, each annotation has a range which includes begin and end of annotation-specific data of that particular annotation. For `Annotation` type, range means the begin index and end index of characters under `Annotation` type in the `text` payload of the `DataPack`. For `AudioAnnotation` type, range means the begin index and end index of sound sample under `AudioAnnotation` type in the `audio` payload of the `DataPack`. For an `Token` instance which is a subtype of `Annotation`, its annotation-specific data is `text` and therefore range means the begin and end of characters of that `Token` instance. For an `Recording` instance which is a subtype of `AudioAnnotation`, its annotation-specific data is `audio` and there range means the begin and end index of that `Recording` instance.
```

Comment:
Here, it is very hard to understand with audio annotation, let's explain this from simple text concepts first.

Reply:
Consider using a visual aid, maybe a picture like this can help you: https://raw.githubusercontent.com/asyml/forte/master/docs/_static/img/forte_results.png